### DISCUSSION TITLE
Checking for active members

### DISCUSSION THREAD

**Discussion by mauricio-lomeli-planview** [react-grid-layout/react-grid-layout#1884] [Q&A] [answered]
@STRML @nikolas @daynin @Semigradsky 
Wanted to ask for a quick simple response. Is anyone still active and able to approve PR contributions? I'm looking into contributing but I'm not sure if anyone will be around to review it.

**Comment by nikolas** [comment]
I don't have any special privileges on this project, I'm just a contributor.

**Comment by STRML** [comment, marked as answer]
Hey, I'm still here. The backlog has really clogged up for a few reasons:

1. I have little time,
2. Many of them need rebasing
3. Others need tests, and
4. I'm not convinced the API of RGL is right. There is too much logic baked into the core that can't be modified by end-users, meaning that everyone tries to extend it with their own logic and add (yet another) option for it. Passing implementations as props and being able to extend those default implementation classes would go a long way, but I have yet to figure out a way to do it cleanly.

If you clearly explain the impetus for the PR and add tests/videos, it goes a long way toward making it easy for me to review & merge.

**Reply by mauricio-lomeli-planview** [reply]
Thank you.
